allure-selenide/src/main/java/io/qameta/allure/selenide/AllureSelenide.java

```diff
@@ -68,6 +81,15 @@ public AllureSelenide savePageSource(final boolean savePageHtml) {
         return this;
     }
 
+    public AllureSelenide saveLogsFrom(String logType) {
+        if (supportedLogTypes.contains(logType)) {
+            logTypesToSave.add(logType);
```

Comment:
you can enable log twice, in that case you'll get duplicate attachments

Reply:
Collection of logs to report was changed to HashMap, so now only one attachment will be reported for each log type.